blog-posts/blog4.qmd

```diff
@@ -4,3 +4,18 @@ description: ""
 date: "11/14/22"
```

Comment:
I can't comment directly on the line, but I'd a bit more to the title and description (if it will show up on socials) here.

**I'm On The Job Market** - *Looking for roles in statistical software.*